pifpaf/drivers/__init__.py

```diff
@@ -201,7 +201,11 @@ def _exec(self, command, stdout=False, ignore_failure=False,
         if stdout or wait_for_line:
             lines = []
             while True:
-                line = c.stdout.readline()
+                try:
+                    line = c.stdout.readline()
+                except KeyboardInterrupt:
```

Comment:
I have wrapped the whole interesting block.

I haven't used finally because we want the process to keep working at the end of _exec()

then the driver call addCleanup(self.kill, pid) and we are ok.